### DISCUSSION TITLE
Wie bildet man einen Handelsregistereintrag

### DISCUSSION THREAD

**Discussion by StephanKuehn** [stephanstapel/ZUGFeRD-csharp#464] [Q&A]
Ich versuche einen Eintrag im Partnerschaftsregister beim Amtsgericht Essen in einer ZUGFeRD 2.3 EN 16931 konformen Rechnung zu kodieren. 

Was wäre denn der korrekte Eintrag bei Seller?

```
SpecifiedLegalOrganization = new LegalOrganization(id: "xxxx", schemaId: ???)
```
Vorab schonmal vielen Dank für Eure Tips!

Viele Grüße
Stephan

**Comment by drocx** [comment]
Ich glaube du kannst dies als Regulatorische Information abbilden, wie in diesem Beispiel:

```
        StringBuilder sellerNote = new();
        sellerNote.AppendLine("Mustermann GmbH");
        sellerNote.AppendLine("Musterweg 11");
        sellerNote.AppendLine("123456 Musterhausen");
        sellerNote.AppendLine("Deutschland");
        sellerNote.AppendLine("Geschäftsführer: Max Mustermann");
        sellerNote.AppendLine("Handelsregisternummer: HRB 12345");
        desc.AddNote(sellerNote.ToString(), SubjectCodes.REG);
```

**Comment by tobitege** [comment]
Hallo @StephanKuehn 
Die Handelsregisternummer kann als ID in BT-30 geschrieben werden (auch ohne schemeID).

Beispiel aus den offiziellen XRechnung 3.0.1 Dateien:
```xml
<ram:SpecifiedLegalOrganization>
  <ram:ID>[HRA-Eintrag]</ram:ID> 
  <ram:TradingBusinessName>[Seller trading name]</ram:TradingBusinessName> 
  </ram:SpecifiedLegalOrganization>
```

Dein Aufruf war also schon richtig.
